language/tools/cost_synthesis/src/stack_generator.rs

```diff
@@ -664,7 +665,8 @@ where
                 };
                 let fields = self
                     .root_module
-                    .field_def_range(num_fields as MemberCount, index);
+                    .field_def_range(num_fields as MemberCount, index)
+                    .expect("[Struct Pack] Module has too many fields");
```

Comment:
I believe you can refer directly to the constructor with the mroe maintainable `rustdoc` syntax, here and in the surroundiing error messages.

Reply:
Forget this, you're just following the error message pattern of the dozen of messages in this file.